commit/src/adapters/extension_mmcs.rs

```diff
@@ -34,6 +34,9 @@ where
     type Error = InnerMmcs::Error;
     type Mat<'a> = ExtensionMatrix<F, EF, InnerMmcs::Mat<'a>> where Self: 'a;
 
+    fn combine(&self, data:&Vec<Self::ProverData>) -> (Self::Commitment, Self::ProverData){
```

Comment:
This produces:

```
warning: function cannot return without recursing
  --> commit/src/adapters/extension_mmcs.rs:37:5
   |
37 |     fn combine(&self, data:&Vec<Self::ProverData>) -> (Self::Commitment, Self::ProverData){
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot return without recursing
38 |     self.combine(data)
   |     ------------------ recursive call site
   |
   = help: a `loop` may express intention better if this is on purpose
   = note: `#[warn(unconditional_recursion)]` on by default
```

Reply:
I think hopefully this is fixed by latest commit